testsuite/MDAnalysisTests/analysis/test_wbridge.py

```diff
@@ -174,23 +574,57 @@ def test_count_by_type_multiple_link(self):
     4ALA      O    5   0.600   0.000   0.000
  1.0   1.0   1.0'''
         u = MDAnalysis.Universe(StringIO(grofile), format='gro')
-        wb = WaterBridgeAnalysis(u, 'protein and (resid 1)', 'protein and (resid 4)')
+        wb = WaterBridgeAnalysis(u, 'protein and (resid 1)', 'protein and (resid 4)', order=4)
         # Build an dummy WaterBridgeAnalysis object for testing
-        wb._timeseries = True
-        wb.timesteps = [0]
-        wb._water_network = [{('SOL', 2): [{(2, 0, ('SOL', 2, 'HW1'), ('ALA', 1, 'O'), 2.0, 179.99998),
-                                            (1, 2, ('ALA', 1, 'H'), ('SOL', 2, 'OW'), 2.0, 179.99998)},
-                                           {(3, 4, ('SOL', 2, 'HW2'), ('ALA', 4, 'O'), 2.0, 179.99998),
-                                            (3, 2, ('ALA', 4, 'H'), ('SOL', 2, 'OW'), 2.0, 179.99998)}]}]
-        result = [(0, 3, 'ALA', 1, 'O', 'ALA', 4, 'H', 1.0),
-                  (0, 4, 'ALA', 1, 'O', 'ALA', 4, 'O', 1.0),
-                  (1, 3, 'ALA', 1, 'H', 'ALA', 4, 'H', 1.0),
-                  (1, 4, 'ALA', 1, 'H', 'ALA', 4, 'O', 1.0)]
-        assert_equal(sorted(wb.count_by_type().tolist()), result)
+        wb._network = []
```

Comment:
Make it a fixture: it can return a list `wb_network` that you can then use for
```python
wb._network.extend(wb_network)
```
Will make the tests cleaner when you separate data from test logic.

Reply:
Done.